File: leetcode_py/Remove Element.py
class Solution:
    # @param {integer[]} nums
    # @param {integer} val
    # @return {integer}
    def removeElement(self, nums, val):
        idx = 0
        for i in nums:
            if i != val:
                nums[idx] = i
                idx += 1
        return idx

File: leetcode_js/Remove Element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    var cur = 0;
    for(var i = 0; i < nums.length; i++){
    	if(nums[i] != val){
    		nums[cur++] = nums[i];
    	}
    }
    return cur;
};